<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUaModbusValueWidget</class>
 <widget class="QWidget" name="QUaModbusValueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="lineTop">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="Line" name="lineBottom">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QUaModbusValueWidgetEdit" name="widgetValueEdit" native="true"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonPerms">
     <property name="text">
      <string>Permissions</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushButtonApply">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacerBottom">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QUaModbusValueWidgetStatus" name="widgetValueStatus" native="true"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="Line" name="lineMiddle">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <spacer name="horizontalSpacerTop">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>288</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUaModbusValueWidgetEdit</class>
   <extends>QWidget</extends>
   <header location="global">QUaModbusValueWidgetEdit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QUaModbusValueWidgetStatus</class>
   <extends>QWidget</extends>
   <header location="global">QUaModbusValueWidgetStatus</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
